modules/oe_theme_content_project/src/Plugin/ExtraField/Display/FundingExtraField.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_content_project\Plugin\ExtraField\Display;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\extra_field\Plugin\ExtraFieldDisplayFormattedBase;
+use Drupal\Core\Url;
+use Drupal\Component\Utility\Unicode;
+use Drupal\Core\Cache\CacheableMetadata;
+
+/**
+ * Extra field displaying Funding and Proposals Project fields.
+ *
+ * @ExtraFieldDisplay(
+ *   id = "oe_theme_content_project_funding",
+ *   label = @Translation("Funding"),
+ *   bundles = {
+ *     "node.oe_project",
+ *   },
+ *   visible = true
+ * )
+ */
+class FundingExtraField extends ExtraFieldDisplayFormattedBase {
+
+  use StringTranslationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(ContentEntityInterface $entity) {
+    $cacheability = new CacheableMetadata();
+    /** @var \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager */
+    $entity_type_manager = \Drupal::getContainer()->get('entity_type.manager');
+    $funding = $entity->get('oe_project_funding_programme')->getIterator();
+    foreach ($funding as $item) {
+      $pattern[] = [
+        '#type' => 'pattern',
+        '#id' => 'list_item',
+        '#fields' => [
+          'meta' => [
+            'label'=> Unicode::strtoupper($item->getFieldDefinition()->getLabel()),
```

Comment:
I think that better to use css to transform text.

Reply:
Changed to use css